tools/loadtesting/README.md

```diff
@@ -0,0 +1,51 @@
+# Mixer load testing tool
+
+This tool is meant for loadtesting mixer APIs in live clusters. It is based on [locust](https://docs.locust.io/en/stable/).
+
+The specific APIs that are called are defined by a [locustfile](https://docs.locust.io/en/stable/writing-a-locustfile.html), located under `tools/loadtesting/locustfiles`. 
+
+## Usage
+
+If the mixer instance to be tested lives as a k8s service that is not exposed to the internet, we can run a containerized version in the same cluster.
+
+1.  Deploy the pod into the same namespace as Mixer.
```

Comment:
mention set the context to the cluster first?

Reply:
Done.